src/network/networkd-link.c

```diff
@@ -199,6 +199,21 @@ static bool link_proxy_arp_enabled(Link *link) {
         return true;
 }
 
+static bool link_ipv6_proxy_ndp_enabled(Link *link) {
```

Comment:
No, afaict. ipv6_proxy_ndp_is_enabled returns true when
* `link->network->ipv6_proxy_ndp == 0`
* `link->network->ipv6_proxy_ndp == 1`

Only in the second case proxy ndp can be said to be enabled.